src/domains/Townhall/TownhallChat/TownhallChatUser.jsx

```diff
@@ -10,7 +10,7 @@ import MessageItemText from '../../../components/Message/MessageItemText';
 import MessageItemAuthor from '../../../components/Message/MessageItemAuthor';
 import MessageItemTimestamp from '../../../components/Message/MessageItemTimestamp';
 import MessageOwnerActions from '../../../components/Message/MessageOwnerActions';
-import ScrollTo from '../../../components/ScrollTo';
+import ScrollTo from '../../../components/ScrollTo/ScrollTo';
```

Comment:
This file shouldn't need to be modified.  I'm guessing this is one of those update all imports thing from vscode?

Reply:
Done, VScode auto changed the imports
Also changed the `../Message/..` because they are in their own folders, not under Message anymore